### PR TITLE
changed chicken repayment calculations logic and cap for november

### PR DESCRIPTION
**Pull request by shejasabin**
- updated the chicken prepayment amount calculations to (credit * healthpathPercentage/100)

To test here: https://telerivet.com/p/8799a79f/service/SV1161debc042a9de0/edit
- enter the account number: 23560396
- choose option 4 for chicken confirmation
- you should receive a message with a new calculation which is 2 possible chickens

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ussd-sms",
-  "version": "1.0.8",
+  "version": "1.0.9",
```
Comment (nyabongo) [inline package.json:3]: @DevFojo @shejasabin  do we need to bump the version if it is not a release?  I think this should be for deployments, whether the weekly or a hotfix

Comment (DevFojo) [reply package.json:3]: No, only for releases and hotfixes

Comment (shejasabin) [reply package.json:3]: This is for the previous release, forgot to update the previous version on develop. @DevFojo should I disregard it for now and update it on the next release?

Comment (DevFojo) [reply package.json:3]: Let's do the update on the next release

Comment (shejasabin) [reply package.json:3]: okay, will do a backmerge as soon as the hotfix is merged

**Review by nyabongo — APPROVED** [review]

**Review by DevFojo — APPROVED** [review]

**Pull request merged**